app/models/user.py

```diff
@@ -178,15 +168,37 @@ def generate_fake(count=100, **kwargs):
 
         seed()
         for i in range(count):
-            u = User(
-                first_name=forgery_py.name.first_name(),
-                last_name=forgery_py.name.last_name(),
-                email=forgery_py.internet.email_address(),
-                password=forgery_py.lorem_ipsum.word(),
-                confirmed=True,
-                role=choice(roles),
-                **kwargs
-            )
+            role = choice(roles)
+            if role.index == 'merchant':
+                u = Merchant(
+                    first_name=forgery_py.name.first_name(),
+                    last_name=forgery_py.name.last_name(),
+                    email=forgery_py.internet.email_address(),
+                    password=forgery_py.lorem_ipsum.word(),
+                    confirmed=True,
+                    role=choice(roles),
+                    **kwargs
+                )
+            elif role.index == 'vendor':
+                u = Vendor(
+                    first_name=forgery_py.name.first_name(),
+                    last_name=forgery_py.name.last_name(),
+                    email=forgery_py.internet.email_address(),
+                    password=forgery_py.lorem_ipsum.word(),
+                    confirmed=True,
+                    role=choice(roles),
+                    **kwargs
+                )
+            else:
```

Comment:
Instead of duplicating the Merchant creation in the `else`, could we just have one `if-else` with one user type created in the `if` and the other created in the `else`?